packages/library/src/commonMain/kotlin/io/realm/internal/SuspendableNotifier.kt

```diff
@@ -34,19 +35,26 @@ import kotlinx.coroutines.withContext
  * inside writes. Users can therefor not register change listeners on live objects, but it is assumed that other
  * layers check that invariant before methods on this class are called.
  */
-internal class SuspendableNotifier(private val owner: Realm, private val dispatcher: CoroutineDispatcher) {
+internal class SuspendableNotifier(
+    private val owner: Realm,
+    private val dispatcher: CoroutineDispatcher
+) {
 
     companion object {
         val NO_OP_NOTIFICATION_TOKEN = object : Cancellable {
-            override fun cancel() { /* Do Nothing */ }
+            override fun cancel() { /* Do Nothing */
+            }
         }
     }
 
     // FIXME Work-around for the global Realm changed listener not working.
     // FIXME Adding extra buffer capacity as we are otherwise never able to emit anything. Needs to
```

Comment:
I guess we can remove the fixme to investigate. Reason for the buffer capacity is in the docs: https://github.com/Kotlin/kotlinx.coroutines/blob/master/kotlinx-coroutines-core/common/src/flow/SharedFlow.kt#L78